doc/filters.texi

```diff
@@ -10785,6 +10785,14 @@ parameter @var{text}.
 
 If both @var{text} and @var{textfile} are specified, an error is thrown.
```

Comment:
to the commit log:  and vf_drawbox

does this filter work without vf_drawbox?

Reply:
yes, but i use the example to show the output with drawbox, would this better or worse?

Reply:
it doesn't matter how the example is, just say it can draw text from detection bounding boxes.

Reply:
Yes, that makes sense, I modified the commit message